src/align.cpp

```diff
@@ -628,10 +628,7 @@ void align_preprocessor(void)
 
       // step past the 'define'
       pc = chunk_get_next_nc(pc);
-      if (pc == nullptr)
-      {
-         break;
-      }
+      // if (pc == nullptr) will never occurs, at least it is a newline
```

Comment:
(FYI, correct grammar would be "occur" :slightly_smiling_face:.)

That said, `echo -n '#define' | ./uncrustify` will trip the `if` you are removing here...